### PR TITLE
Mechanical Design Tips

### PR DESCRIPTION
**Pull request by aharter23**

### CODE REVIEW

**Comment by benbo** [comment]
Just my opinion, but I'm not convinced that these articles are a good fit for the wiki in their current state. The purpose of the wiki is to advance knowledge in robotics. The contribution of these files here is that they are short introductions and summaries to the links that they include, rather than articles that are of value by themselves. Then again, these might be good starting points for more detailed and elaborate entries.

File: wiki/system-design-development/electronic-and-actuator-selection.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Title goes here
```
Comment (smbryan) [inline wiki/system-design-development/electronic-and-actuator-selection.md]: Missing something.

File: wiki/system-design-development/rapid-prototyping-tips.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Title goes here
```
Comment (smbryan) [inline wiki/system-design-development/rapid-prototyping-tips.md]: Missing title.

File: wiki/system-design-development/electronic-and-actuator-selection.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+published: true
+---
+## Summary
+DC Motor Selection - Link 1
```
Comment (smbryan) [inline wiki/system-design-development/electronic-and-actuator-selection.md]: This isn't our format. Please see [the template](https://github.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/blob/master/_templates/template.md) for additional instructions.

**Comment by smbryan** [comment]
@aharter23, this is a good start but these need some work.  Please review @benbo and my comments and resubmit.

**Pull request closed without merging**